config.sample.yaml

```diff
@@ -4,5 +4,3 @@ location: __CHANGE_ME__
 environment: __CHANGE_ME__
 arm_subscription_id: __CHANGE_ME__
 devcontainer_acr_name: __CHANGE_ME__
-transform:
- spark_version: 3.3.1
```

Comment:
did you want to add the installing the right spark version into the devcontainer as a part of this PR?

Reply:
I didn't _want to_, but I did :)